java-chassis-dependencies-springboot2/pom.xml

```diff
@@ -37,10 +37,11 @@
     <argLine>-Dfile.encoding=UTF-8</argLine>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <spring.version>5.0.4.RELEASE</spring.version>
-    <spring.boot.version>2.0.0.RELEASE</spring.boot.version>
-    <spring.data.version>2.0.5.RELEASE</spring.data.version>
+    <spring.version>5.0.6.RELEASE</spring.version>
+    <spring.boot.version>2.0.3.RELEASE</spring.boot.version>
+    <spring.data.version>2.0.3.RELEASE</spring.data.version>
     <spring.session.version>2.0.3.RELEASE</spring.session.version>
+    <spring-cloud.version>Finchley.RELEASE</spring-cloud.version>
   </properties>
 
   <dependencyManagement>
```

Comment:
<dependencies>
       <dependency>	       <dependency>
         <groupId>org.springframework.boot</groupId>	         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter</artifactId>	         <artifactId>spring-boot-starter</artifactId>
         <version>${spring.boot.version}</version>	         <version>${spring.boot.version}</version>
         <type>pom</type>	         <type>pom</type>
         <scope>import</scope>	         <scope>import</scope>
       </dependency>	       </dependency>
       <dependency>	       <dependency>
         <groupId>org.springframework.boot</groupId>	         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-web</artifactId>	         <artifactId>spring-boot-starter-web</artifactId>
         <version>${spring.boot.version}</version>	         <version>${spring.boot.version}</version>
         <type>pom</type>	         <type>pom</type>
         <scope>import</scope>	         <scope>import</scope>
       </dependency>	       </dependency>

Can be deleted if using spring boot dependencies.

Reply:
All redundant dependencies were cleaned up.